languages.toml

```diff
@@ -165,6 +165,31 @@ roots = []
 
 indent = { tab-width = 4, unit = "\t" }
 
+[[language]]
+name = "julia"
+scope = "source.julia"
+injection-regex = "julia"
+file-types = ["jl"]
+roots = []
+
+args = [
+        "--startup-file=no",
+        "--history-file=no",
+        "-e",
+        """
+                using LanguageServer;
+                using Pkg;
+                import StaticLint;
+                import SymbolServer;
+                env_path = dirname(Pkg.Types.Context().env.project_file);
+
+                server = LanguageServer.LanguageServerInstance(stdin, stdout, env_path, "");
+                server.runlinter = true;
+                run(server);
+        """,
+        ]
```

Comment:
Is all these arguments necessary?

Reply:
Not sure… 
As I said, I copied it from https://github.com/julia-vscode/LanguageServer.jl/wiki/Kakoune, but since the language server for julia is not a distinct executable but meant to be used from a julia session, I think it pretty much _is_ necessary.